### PR TITLE
Some little cleanups

### PR DESCRIPTION
**Pull request by gkoz**
I like deleting code and didn't feel like doing serious stuff.. Hope you find this useful.

I think there was a sorting bug hiding somewhere in those comparators. It's good they aren't really necessary. Copying each string twice for each comparison wasn't a nice pattern.

### CODE REVIEW

File: src/process_viewer.rs [hunk 1]
```diff
@@ -155,76 +123,74 @@ impl Procs {
...
+    let mut seen: HashSet<usize> = HashSet::new();
+
+    if let Some(mut iter) = list.get_iter_first() {
+        let mut valid = true;
+        while valid {
+            let pid = list.get_value(&iter, 0).get::<i64>().unwrap() as usize;
+            if let Some(p) = entries.get(&(pid)) {
+                list.set(&iter,
+                         &[2, 3, 5],
+                         &[&format!("{:.1}", p.cpu_usage), &p.memory, &p.cpu_usage]);
```
Comment (GuillaumeGomez) [inline src/process_viewer.rs]: The last column isn't cpu.

File: src/process_viewer.rs [hunk 2]
```diff
@@ -107,11 +70,20 @@ impl Procs {
         append_column("process name", &mut columns, &left_tree);
         append_column("cpu usage", &mut columns, &left_tree);
         append_column("memory usage (in kB)", &mut columns, &left_tree);
-
-        let mut list_store = gtk::ListStore::new(&[glib::Type::I64, glib::Type::String,
-                                                   glib::Type::String, glib::Type::U32]);
+        // sort the name column by name_lowercase
+        columns[1].set_sort_column_id(4);
```
Comment (GuillaumeGomez) [inline src/process_viewer.rs]: How does it work then?

**Comment by GuillaumeGomez** [comment]
That's... Wonderful! I like when you don't do "serious" stuff haha.

**Comment by gkoz** [comment]
I decided to take advantage of `TreeViewColumn` allowing to assign any columns as sorting keys. The two new columns are invisible, they only exist in the model but thanks to them we don't need to transform values in columns `1` and `2` to compare them, but put ready to compare data into the invisible ones. Ideally the CPU_f32 column would not be needed but I don't know how to format float nicely in the renderer instead of when `insert`ing or `set`ting. We could be missing some bindings.

**Comment by GuillaumeGomez** [comment]
In this case, can you add comments please?

**Comment by gkoz** [comment]
Updated

**Comment by GuillaumeGomez** [comment]
Nice, thanks a lot! Waiting for travis and we're good to go! :)

**Pull request merged**